rasa/utils/common.py

```diff
@@ -385,6 +385,20 @@ def formatwarning(
     warnings.formatwarning = original_formatter
 
 
+def raise_deprecation_warning(
+    message: Text, warn_until_version: Text, docs: Optional[Text] = None, **kwargs: Any,
+) -> None:
+    """
+    Thin wrapper around `raise_warning()` to raise a deprecation warning. It requires
+    a version until which we'll warn, and after which the support for the feature will
+    be removed.
+    """
+    if warn_until_version not in message:
+        message = f"{message} (will be removed in {warn_until_version})"
+
+    raise_warning(message, DeprecationWarning, docs, **kwargs)
```

Comment:
If we don't raise a `FutureWarning`, it won't be printed into the user's console.

Reply:
arf, but why?

Reply:
https://docs.python.org/3/library/warnings.html#warning-categories

Deprecation warning: 

Base category for warnings about deprecated features when those warnings are intended for other Python developers (ignored by default, unless triggered by code in __main__).
--


Furue warning: 

Base category for warnings about deprecated features when those warnings are intended for end users of applications that are written in Python.
--
 
Not sure if things are triggered by code in main? but would need to check. this was implemented because important messages werent getting to the user 🤔

Reply:
ok will test